plugins/tenant/acme/config/tenant.yml

```diff
@@ -0,0 +1,39 @@
+default: &default
+  name: 'ACME Üniversitesi'
+  abbreviation: 'acme'
+  api_host: 'https://api.omu.sh'
+  contact:
+    address: 'ACME Üniversitesi, Kurupelit Kampüsü, Atakum/SAMSUN'
+    phone: '+9 0362 312 1919'
+    country: 'TR'
+    district: 'Atakum'
+  email:
+    domain: 'omu.edu.tr'
```

Comment:
Yol üzerinde bu referansları da değiştimemiz lazım (geleceğe yönelik not), `acme.omu.sh` gibi...

Reply:
Bu kısmı biraz düşüneyim hocam. Etkilenecek yerler olabilir.